File: RxJavaRetrofit/app/src/main/java/com/woowrale/rxjavaretrofit/ui/main/MainView.java
package com.woowrale.rxjavaretrofit.ui.main;

import com.woowrale.rxjavaretrofit.data.model.Crypto;
import com.woowrale.rxjavaretrofit.ui.base.BasePresenter;

import java.util.List;

public interface MainView extends BasePresenter.View {

    void showListView(List<Crypto.Market> marketList);

}

File: RxDiKotlinRetrofit/app/src/main/java/com/woowrale/rxkotlinretrofit/ui/main/MainView.kt
package com.woowrale.rxkotlinretrofit.ui.main

import com.woowrale.rxkotlinretrofit.data.model.Market
import com.woowrale.rxkotlinretrofit.ui.base.BasePresenter

interface MainView: BasePresenter.View {
    fun setRecyclerViewAdapterData(marketList: List<Market>)
}